client/landing/gutenboarding/components/header/style.scss

```
@@ -14,7 +14,7 @@ $padding--gutenboarding__header: $grid-unit-10;
 	border-bottom: 1px solid $light-gray-500;
 	height: $gutenboarding-header-height;
 	background: $white;
-	z-index: z-index( '.block-editor-editor-skeleton__header' );
+	z-index: 1;
```

Let's avoid hardcoding :grimacing: 

```suggestion
	z-index: z-index( '.interface-interface-skeleton__header' );
```


These `z-index` changes normally come from Gutenberg renaming selectors, e.g. when moving a component to a different package. To find the relevant change, check the commit history of https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_z-index.scss.

In this case, the relevant change was here: https://github.com/WordPress/gutenberg/pull/21335/files#diff-1c24406914c18d0668eb1f64880d4144R27